### PR TITLE
[CFM-1027-fix] Change notification mails body format from "full_html" to "mail'.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-1027

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 1]
```diff
@@ -858,8 +880,9 @@ function c4m_message_default_message_type() {
...
-          "format" : "full_html"
+          "value" : "\\u003Cp\\u003EDear @member_full_name,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E\\u003Cbr \\/\\u003E\\r\\nThank you for requesting membership to @group_node_title at [site:name].\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYour membership is currently waiting to be approved by one of the @group_node_type\\u2019s administrators, and should be processed shortly.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EIf you have any questions, please feel free to \\u003Ca href=\\u0022mailto:capacity4dev@ec.europa.eu\\u0022\\u003Econtact us\\u003C\\/a\\u003E; we will be happy to help.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EThe [site:name] team\\u003C\\/p\\u003E\\r\\n",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: @anvmn 
why do we need to change the original value in here ?

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 2]
```diff
@@ -780,8 +799,9 @@ function c4m_message_default_message_type() {
...
-          "format" : "full_html"
+          "value" : "\\u003Cp\\u003EDear @user_full_name,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E\\u003Ca href=\\u0022!event_user_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@event_user_full_name\\u003C\\/a\\u003E edited the following @content_type_name: \\u003Cstrong\\u003E\\u003Ca href=\\u0022!content_node_url\\u0022\\u003E@content_node_title\\u003C\\/a\\u003E\\u003C\\/strong\\u003E\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003ETake a look at the latest changes by accessing this @content_type_name\\u003Cbr \\/\\u003E\\r\\n\\u003Ca href=\\u0022!content_node_url\\u0022\\u003ERead more...\\u003C\\/a\\u003E\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYou\\u0027re receiving this e-mail because you\\u0027re subscribed to the @content_type_name \\u003Ca href=\\u0022!content_node_url\\u0022\\u003E@content_node_title\\u003C\\/a\\u003E\\u0026nbsp;- if you no longer wish to receive notifications from this @content_type_name, please \\u003Ca href=\\u0022!unsubscribe_content_url\\u0022\\u003Eunsubscribe.\\u003C\\/a\\u003E\\u003C\\/p\\u003E\\r\\n",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: @anvmn 
why do we need to change the original value in here ?

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 3]
```diff
@@ -598,8 +610,9 @@ function c4m_message_default_message_type() {
...
-          "format" : "full_html"
+          "value" : "\\u003Cp\\u003EDear colleague,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E\\u003Cbr \\/\\u003E\\r\\nThank you for your interest in the group \\u003Cstrong\\u003E\\u003Ca href=\\u0022!group_dashboard_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@group_node_title\\u003C\\/a\\u003E\\u003C\\/strong\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EUnfortunately we cannot accept your membership request because @membership_reject_reason.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EThank you for your understanding.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EDon\\u0027t hesitate to contact me if your have further questions.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E\\u003Cbr \\/\\u003E\\r\\n\\u003Ca href=\\u0022!event_user_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@event_user_full_name\\u003C\\/a\\u003E,\\u003Cbr \\/\\u003E\\r\\n@group_node_type Administrator - @group_node_title\\u003C\\/p\\u003E\\r\\n",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: @anvmn 
why do we need to change the original value in here ?

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 4]
```diff
@@ -277,8 +286,9 @@ function c4m_message_default_message_type() {
...
-          "format" : "full_html"
+          "value" : "\\u003Cp\\u003EDear @group_owner_full_name,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E\\u003Cbr \\/\\u003E\\r\\nThank you for requesting the creation of the following @group_node_type: @group_node_title at [site:name].\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYour request is currently pending review by one of the site\\u0027s administrators and should be treated shortly.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EIf you have any questions, please feel free to \\u003Ca href=\\u0022mailto:capacity4dev@ec.europa.eu\\u0022 rel=\\u0022nofollow\\u0022\\u003Econtact us\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EThe [site:name] team\\u003C\\/p\\u003E\\r\\n",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: @anvmn 
why do we need to change the original value in here ?

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 5]
```diff
@@ -225,8 +232,9 @@ function c4m_message_default_message_type() {
...
...\\u003E@group_node_title\\u003C\\/strong\\u003E was approved and is currently saved in draft status.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E\\u003Cbr \\/\\u003E\\r\\n\\u003Cem\\u003E@admin_comment \\u003C\\/em\\u003E\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYou can access your new \\u003Ca href=\\u0022!group_dashboard_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@group_node_type here\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EAs the owner, we encourage you to read \\u003Ca href=\\u0022!help_section_url\\u0022 rel=\\u0022nofollow\\u0022\\u003Ethis short guide\\u003C\\/a\\u003E. Here you\\u2019ll find tips and advice to make the most of your @group_node_type. Once you\\u0027re ready to go live, don\\u0027t forget to update the status of your @group_node_type to share it with the rest of the community.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EWe hope you\\u2019ll enjoy using [site:name] and we\\u2019d like to remind you that we are available to provide \\u003Ca href=\\u0022@site_contact_link\\u0022 rel=\\u0022nofollow\\u0022\\u003Ehelp and support\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EThe [site:name] team\\u003C\\/p\\u003E\\r\\n",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: @anvmn 
why do we need to change the original value in here ?

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc [hunk 6]
```diff
@@ -69,8 +70,9 @@ function c4m_message_default_message_type() {
...
-          "format" : "filtered_html"
+          "value" : "\\u003Cp\\u003EDear @member_full_name,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYour administration rights for the @group_node_type \\u003Ca href=\\u0022!group_dashboard_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@group_node_title\\u003C\\/a\\u003E have been removed.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EIf you have any questions about your new role within this @group_node_type, please feel free to \\u003Ca href=\\u0022!event_user_message_url\\u0022 rel=\\u0022nofollow\\u0022\\u003Econtact me\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E@event_user_full_name\\u003Cbr \\/\\u003E\\r\\n@group_node_type Administrator, @group_node_title\\u003C\\/p\\u003E\\r\\n",
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: @anvmn 
why do we need to change the original value in here ?

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc]: This is what Drupal has decided.
I made the change through UI, and then exported the feature.

**Comment by anvmn** [comment]
Travis fails due to issue reported at https://issuetracker.amplexor.com/jira/browse/CFM-422:

selection_445

**Pull request merged**